src/quantile.jl

```diff
@@ -1,6 +1,14 @@
 """
-    quantile(var, design, q)
-Estimate quantiles for `SurveyDesign`s.
+    quantile(var, design, q; kwargs...)
```

Comment:
```suggestion
    quantile(var, design, q; kwargs...)
    
```

Reply:
I have updated this. To keep with `Statistics` and `Statsbase` syntax, we should use `p` instead of `q` for the vector of probabilities